### ISSUE TITLE
Incompatibility with trio v0.11.0

### ISSUE THREAD

**Issue by alidursen** [HyperionGray/trio-websocket#102]
My very mundane Dockerfile
```Dockerfile
FROM python:3.5-slim
RUN apt-get update && apt-get install -y gcc # Since trio-ws needs it
RUN python3 -m pip install trio-websocket # Since it installs trio by itself
CMD ["python3", "-cimport trio_websocket\nprint(\"exiting\")"]
```
builds fine but gives an interesting error when run:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.5/site-packages/trio_websocket/__init__.py", line 1, in <module>
    from ._impl import (
  File "/usr/local/lib/python3.5/site-packages/trio_websocket/_impl.py", line 13, in <module>
    import trio.ssl
ImportError: No module named 'trio.ssl'
``` 
I can `import trio_websocket` normally just fine. I am at a loss.

**Comment by njsmith** [comment]
Ah crud. This is a subtle incompatibility introduced in trio v0.11.0 (released yesterday). We deprecated the `trio.ssl` module. In order to issue a deprecation warning when people access `trio.ssl`, we converted the `trio.ssl` attribute into a special property, that issues the warning and then returns the module object. Which is supposed to be backwards compatible. But... apparently `import` statements don't know how to handle this special magic, so we broke `import trio.ssl`.
